bigquery/bigquery_reservation_quickstart/main.go

```diff
@@ -26,48 +26,57 @@ import (
 	"fmt"
 	"log"
 
-	bqReservation "cloud.google.com/go/bigquery/reservation/apiv1"
+	reservationapi "cloud.google.com/go/bigquery/reservation/apiv1"
 	"google.golang.org/api/iterator"
 	reservationpb "google.golang.org/genproto/googleapis/cloud/bigquery/reservation/v1"
 )
 
 // Define two command line flags for controlling the behavior of this quickstart.
 var (
-	projectID = flag.String("project_id", "",
-		"Cloud Project ID, used for session creation.")
-	location = flag.String("location", "US",
-		"BigQuery location used for interactions")
+	projectID = flag.String("project_id", "", "Cloud Project ID, used for session creation.")
+	location  = flag.String("location", "US", "BigQuery location used for interactions")
```

Comment:
Would you mind moving these declarations inside `func main`?

Reply:
done